themes/default/content/docs/support/troubleshooting.md

```diff
@@ -196,6 +196,24 @@ You may encounter an error when you downgrade provider versions _after_ your sta
 If you must downgrade the version of a provider your `pulumi` program depends on, you will need to [manually edit your deployment](#editing-your-deployment)
 and change the version of the provider your stack depends on and then import that as the latest state of your stack.
 
+### Nothing happens when running Pulumi
```

Comment:
can we add proxy to this heading somehow?

Reply:
I will review it later today. 😉

Reply:
Updated the title.